### ISSUE TITLE
E711/E712 not correcting

### ISSUE THREAD

**Issue by graingert** [hhatto/autopep8#489]
## Python Code
```python
h = None
assert None == hj
```

Command Line
```shell
$ autopep8 --in-place -aaaaaa --ignore-local-config --select E711,E712 -r foo.py 
```

I expected to get

```python
h = None
assert None is h
```

**Comment by Hanaasagi** [comment]
```Python
h = None
assert None == h
assert h == None
```

After formatted by`autopep8 --aggressive t.py`.

```Python
h = None
assert None == h
assert h is None
```

It will not work when `None` is a left operand. Confirmed and briefly checked the code.

https://github.com/hhatto/autopep8/blob/fff90d479862f65ac57678f150a97217439c1bc9/autopep8.py#L1067-L1068

It just return :flushed:.

**Comment by hhatto** [comment]
this is fix in #498 .

I plan release to version 1.5.
